### ISSUE TITLE
Request: Smaller slides (option or another view) to give fanart a bit more visibility

### ISSUE THREAD

**Issue by olavinto** [jeroenpardon/skin.grid#162]
### Description

Would it be possible to get an optional view where the slides are smaller. For poster views this would be especially nice because currently they take up most of the screen and while they look nice they hide the background fan art almost completely. No means do I want the original view to be edited or replaced but an option for it or another view with like ~40% smaller posters would be very nice.

### Checklist:

- [X] I have checked if I am using the latest version of Grid at the time of opening this issue
- [X] I am running a Kodi version that is listed as compatible in Grid's README file
- [X] I have checked if all of the [required add-ons](https://github.com/jeroenpardon/skin.grid.beta/wiki/Required-and-supported-add-ons) are installed on my system
- [X] I have checked whether this behaviour is also present in Kodi's default skin
- [ ] I have screenshots or a video for further explanation. I have included a link to the image(s) or video in my description 
- [ ] I have a debug log for (further) explanation. I have included a link to the log on pastebin.com or similar service in my description

**Comment by jeroenpardon** [comment]
This will probably happen at some point, but it's low priority at the moment tbh. And if it does I doubt it would adress your wish thoug, as it would mean the space in between the posters wouldn't necessarily be bigger.

**Comment by olavinto** [comment]
Yeah, but it would be enough to get more free space on top of the poster row so that the backdrop would show up more as currently the central part (which is usually the most notable) is pretty much completely covered by the poster images. Even on TV episode lists it is already better because the landscape episode thumbs free up the top portion of the view. Showcase views on many skins are a good example of what I'd like and they also have the text part on top of the posters so that only the bottom part of the background is covered - maybe this could also work for GRiD. But you do what you will, it's great even as it is. :-)
